vcpkg/users/platforms/android.md

```diff
@@ -40,24 +33,41 @@ There are four different Android ABIs, each of which maps to a vcpkg triplet. Th
 |---------------------------|----------------------|
 |arm64-android              | arm64-v8a            |
 |arm-android                | armeabi-v7a          |
+|arm-neon-android (w/neon)  | armeabi-v7a          |
 |x64-android                | x86_64               |
 |x86-android                | x86                  |
+|armv6-android              | armeabi              |
 
 ## Install libraries for Android using vcpkg
 
 Example for jsoncpp:
 
 ```bash
-cd $VCPKG_ROOT
-
-# specify the triplet like this
 ./vcpkg install jsoncpp --triplet arm-android
 # or like this
 ./vcpkg install jsoncpp:arm64-android
 ./vcpkg install jsoncpp:x86-android
 ./vcpkg install jsoncpp:x64-android
 ```
 
+## Building Android libraries in a Docker container
+
+You can also build Android libraries in a Ubuntu Docker container (same environment as our CI).
```

Comment:
I don't think we should document a subpath into our CI scripts here (that we're likely to move!) -- can we trim down our Dockerfile and include it inline here (since docs readers aren't building the universe)?